index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav class="nav-container">
+            <ul class="nav-links">
+                <li class="logo" >
+                    <a href="#">My logo</a>
+                </li>
+                <li class="bars">
+                    <a href="#"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></a>
+                </li>
+            </ul>
+    </nav>
+    <main class="main-container">
+        <div class="flex-name">
+            <h1>I'm Kwamboka Glad to see you!</h1>
+        </div>
+        <div class="flex-skills">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
```

Comment:
- [ ] Please place some top padding to the hero section and make sure it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158961898-021270e6-cd38-4152-8574-bf3a8d81f3bf.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/158961963-379ac00b-f175-4e56-8123-7bde4846b2bc.png)